README.md

```diff
@@ -335,3 +335,74 @@ The response contains the same data as when the repair was created:
 
 As above: if the repair was created without work orders, an empty array of
 workOrders should be returned (rather than no key or a null value)
+
+
+## Get Appointments for Work Order
+
+Returns a list of appointments matching the given criteria.
+
+```
+GET /v1/appointments/
```

Comment:
The Work Order Reference is a fundamental part of this request - not a search parameter (as was the case with list properties) - so it should be part of the URL: `/v1/appointments/work_order_reference`.

Note that while in text we're using mixed-case so we are using`camelCaseWords`, in URLs we should always always always use lower case, but we still need to separate the words so that they are easier to read, so we use `snake_case_words`.